packages/mdc-textfield/_mixins.scss

```diff
@@ -180,6 +180,132 @@
   }
 }
 
+@mixin mdc-text-field-outlined-focused_ {
+  .mdc-text-field__idle-outline {
+    opacity: 0;
+  }
+
+  .mdc-text-field__outline {
+    opacity: 1;
+  }
+
+  .mdc-text-field__outline-path {
+    stroke-width: 2px;
+  }
+}
+
+@mixin mdc-text-field-outlined-dense_ {
+  height: 48px;
+
+  .mdc-text-field__input {
+    padding: 12px 12px 7px;
+  }
+
+  .mdc-text-field__label {
+    bottom: 18px;
+
+    &--float-above {
+      transform: translateY(calc(-122% - 2px)) scale(.923);
+
+      // stylelint-disable max-nesting-depth
+      // stylelint-disable-next-line selector-max-specificity
+      &.mdc-text-field__label--shake {
+        animation: invalid-shake-float-above-outlined-dense 250ms 1;
+      }
+      // stylelint-enable max-nesting-depth
+    }
+  }
+
+  .mdc-text-field__icon {
+    top: 12px;
+  }
+}
+
+@mixin mdc-text-field-outlined_ {
+  height: 56px;
+  border: none;
+
+  // stylelint-disable plugin/selector-bem-pattern
+
+  .mdc-text-field__input {
+    display: flex;
+    height: 30px;
+    padding: 12px;
+    border: none;
+    background-color: transparent;
+    z-index: 1;
+
+    &:hover ~ .mdc-text-field__idle-outline {
+      border: 1px solid;
+    }
+  }
+
+  .mdc-text-field__label {
+    @include mdc-rtl-reflexive-position(left, 16px);
+
+    position: absolute;
+    bottom: 20px;
+    transition: transform 260ms ease;
+
+    &--float-above {
+      transform: scale(.75) translateY(-170%);
+
+      &.mdc-text-field__label--shake {
+        animation: invalid-shake-float-above-outlined 250ms 1;
+      }
+    }
+  }
+
+  .mdc-text-field__icon {
+    z-index: 2;
+
+    &:hover ~ .mdc-text-field__idle-outline {
+      border: 1px solid $mdc-text-field-outlined-hover-border;
+    }
+  }
+
+  &.mdc-text-field--focused {
```

Comment:
Don't put this within the`mdc-text-field-outlined` mixin, just call it directly from `mdc-text-field.scss`